irohad/ordering/impl/ordering_gate_cache/on_demand_cache.hpp

```diff
@@ -18,7 +18,14 @@ namespace iroha {
 
       class OnDemandCache : public OrderingGateCache {
        public:
-        void addToBack(const BatchesSetType &batches) override;
+        /**
+         * Create OnDemandCache instance
+         * @param max_cache_size - maximum amount of transactions contained in
+         * cache (calculated as a sum over all stored batches)
```

Comment:
in cache -> in a single cache entry, since there are 3 entries, and the restriction is for a single entry.

Reply:
discussed offline: not a subject of change